ams/dist/src/main/arctic-bin/conf/config.yaml

```diff
@@ -17,6 +17,10 @@ ams:
   http-server:
     bind-port: 1630
 
+  iceberg:
+    worker:
+      num-threads: 20
+
```

Comment:
Should we directly introduce a configuration domain for system properties, instead of designing a separate configuration parameter for this? @zhoujinsong

Reply:
I don't think the goal of this configuration is just to set an iceberg-specific system variable.
It should be a general configuration that applies to all table formats, more like `table-manifest-io.thread-count` introduced by #2116.

Maybe we should reuse the configuration and set the value to the system variable `iceberg.worker.num-threads` for now.

HDYT? @zhongqishang

Reply:
Yes, I agree. we can reuse configuration `table-manifest-io.thread-count`, and also reuse `ThreadPools.getWorkerPool()`.